src/sentry/models/project.py

```diff
@@ -98,6 +98,7 @@ class Project(Model):
         'pk',
         'slug',
     ])
+    platform = models.CharField(max_length=64, null=True)
```

Comment:
fwiw matt we had 244 new signups in the last day so ~10/hr if thats helpful with the tiny blip analysis